torchrl/envs/transforms/transforms.py

```diff
@@ -312,6 +312,8 @@ def __init__(
         device = kwargs["device"]
         super().__init__(**kwargs)
         self._set_env(env, device)
+        self._batch_locked = env._batch_locked
```

Comment:
I don't think it's appropriate to put this in `__init__`

Reply:
Yes i agree i didn't really now how to set it since we only get the env info in the init but _new_ is called before. How do you think we should proceed?

Reply:
oh yeah you're right
what about

```
@property
def batch_locked(self):
    return self.base_env.batch_locked
```

We should also consider what to do with `_BatchedEnv`

Reply:
Ok but we still need to overwrite _batch_locked, no? We don't want it to keep stuck at True if some reason we re-use the attribute

Reply:
what do you mean? That a transformed env that has a batch_locked base env could itself be non batched_locked?

Reply:
Ok i didn't understood what you said. What i mean is that _batch_locked will always be True for transformed env

Reply:
Oh why?

Reply:
I've just realized that this is not an issue anymore with the parameter passing in _new_. Was an issue before if we did
```
@property
def batch_locked(self):
    return self.base_env.batch_locked
```